.github/workflows/build-container.yaml

```diff
@@ -0,0 +1,47 @@
+name: Building and publishing container
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*.*.*"
```

Comment:
Should the action be constrained like this or build on PRs as well.
Also I still need to figure out how to propagate the tag.

Reply:
> Should the action be constrained like this or build on PRs as well.

Is it possible to look for one of two tags? We can keep the `v*.*.*` tag, but add another one that's something like `force-docker-build` in the event we want some specific PR to update the image.

> Also I still need to figure out how to propagate the tag.

I'm not sure what you mean by this.

Reply:
That the container we build off a tagged commit gets that same tag on dockerhub